src/server/pfs/server/http.go

```diff
@@ -107,27 +114,54 @@ type loginRequestPayload struct {
 }
 
 func (s *HTTPServer) authLoginHandler(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
-	var data loginRequestPayload
-	decoder := json.NewDecoder(r.Body)
-	err := decoder.Decode(&data)
-	if err != nil {
-		// Return 500
-		http.Error(w, fmt.Sprintf("malformed JSON sent in payload: %v", err), http.StatusInternalServerError)
-		return
-	}
-	if data.Token == "" {
+	token := r.FormValue("Token")
+	if token == "" {
 		// Return 500
 		http.Error(w, "empty token provided", http.StatusInternalServerError)
 		return
 	}
-	w.Header().Add("Set-Cookie", fmt.Sprintf("%v=%v", auth.ContextTokenKey,
-		data.Token))
+	w.Header().Add("Set-Cookie", fmt.Sprintf("%v=%v;path=/", auth.ContextTokenKey,
```

Comment:
Note for posterity: Cookies set by `/auth/login` don't apply to `/pfs/...` unless you set `path=/` in the response